test/cql-pytest/test_permissions.py

```diff
@@ -553,3 +554,174 @@ def test_native_functions_always_exeutable(cql):
                     assert list(user_session.execute(f"SELECT max(a) FROM {table}")) == [(84,)]
                     assert list(user_session.execute(f"SELECT min(a) FROM {table}")) == [(3,)]
                     assert list(user_session.execute(f"SELECT sum(a) FROM {table}")) == [(102,)]
+
+# When a role has permissions to create a table, the role is given full
+# access to the table it just created (we call this feature "auto-grant").
+# The following tests check the auto-grant feature. The first test checks
+# that the base table created by CREATE TABLE is usable by the role creating
+# it. The second test is about the view created by CREATE MATERIALIZED VIEW
+# being accessible. The third test is about newly created CDC being accessible.
```

Comment:
Maybe better to keep comments about other testcases close to these testcases, because e.g. what if someone inserts a testcase in the middle - it'd invalidate this comment.

Reply:
I'll add small comments next to each small test, but I do think that a big comment before a "section" of tests is also a good idea (this is one of the reasons why I don't like "docstrings" and prefer comments) so I prefer to keep both the big comment and the smaller ones before each test function.
In this case, the big comment introduces **why** we need these "auto_revoke" tests in the first place, and then we have three tests for this plus one utility function.